app/src/pages/inside/stepPage/retries/stackTrace/emptyMessage/emptyStackTrace.jsx

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+import Parser from 'html-react-parser';
+import ErrorIcon from 'common/img/error-inline.svg';
+import styles from './emptyStackTrace.scss';
+
+const cx = classNames.bind(styles);
+
+export const EmptyStackTrace = () => (
```

Comment:
Can you use existed component for this from pages/inside/logsPage/logItemInfo/logItemInfoTabs/noItemMessage (Just make a rebase - now this component located at components/main/noItemMessage)?